### PR TITLE
Release 22.05

### PR DESCRIPTION
**Pull request by ax3l**
Prepare the May release of WarpX:
```bash
# update dependencies
./Tools/Release/updateAMReX.py
./Tools/Release/updatePICSAR.py
# bump version number
./Tools/Release/newVersion.sh
```

Following this workflow: https://warpx.readthedocs.io/en/latest/maintenance/release.html

### CODE REVIEW

**Comment by ax3l** [comment]
Note: will close & re-open this PR (to trigger CI) once AMReX 22.05 is released :)

**Comment by WeiqunZhang** [comment]
AMReX 22.05 has been released.

**Review by WeiqunZhang — APPROVED** [review]

**Comment by RemiLehe** [comment]
@WeiqunZhang 
It seems that the automated tests are failing for RZ electrostatic solves, with the following error:
```
Assertion `m_sigma[0] == 0._rt' failed, file "/tmp/ci/warpx/builddir/_deps/fetchedamrex-src/Src/LinearSolvers/MLMG/AMReX_MLEBNodeFDLaplacian.cpp", line 313, Msg: "r-direction sigma must be zero" !!!
```
I am guessing that this means we need to call `linop.setSigma(` in the RZ case and set the first component to 0.
I will go ahead and modify this PR accordingly.

**Comment by WeiqunZhang** [comment]
Oh.  That assertion is wrong.  It should be `== 1` not `== 0`.

**Comment by RemiLehe** [comment]
Note that this minimal fix will then be cleaned up by this PR: 

https://github.com/ECP-WarpX/WarpX/pull/3026

**Comment by RemiLehe** [comment]
Note that this minimal fix will then be cleaned up by this PR: 

https://github.com/ECP-WarpX/WarpX/pull/3026/files

**Comment by RemiLehe** [comment]
@WeiqunZhang Oh, ok. I guess this will be part of the next `amrex` release, correct? 
For now, I will still set `sigma` to 0 for the RZ case.

**Comment by RemiLehe** [comment]
@WeiqunZhang Oh, ok. I guess this will be part of the next `amrex` release, correct? 
For now, I will still set `sigma` to 0 for the RZ case.

**Comment by WeiqunZhang** [comment]
OK.  Maybe I will simply remove the assertion.

File: Source/FieldSolver/ElectrostaticSolver.cpp [hunk 1]
```diff
@@ -370,7 +370,9 @@ WarpX::computePhi (const amrex::Vector<std::unique_ptr<amrex::MultiFab> >& rho,
         // With embedded boundary: extract EB info
         MLEBNodeFDLaplacian linop( {Geom(lev)}, {boxArray(lev)}, {DistributionMap(lev)}, info, {&WarpX::fieldEBFactory(lev)});
 
-#ifndef WARPX_DIM_RZ
+#ifdef WARPX_DIM_RZ
+            linop.setSigma(0._rt, 1._rt);
```
Comment (ax3l) [inline Source/FieldSolver/ElectrostaticSolver.cpp]: Fixed syntax to address compilation error:
```suggestion
            linop.setSigma({0._rt, 1._rt});
```
- [ ] testing locally

**Pull request merged**